### PR TITLE
Add documentation from archive repo

### PR DESCRIPTION
**Pull request by bemwen**
## Description

In the previous repositiory which was used before splitting out into domains, we had a number of documents that should now instead be living in this repositity and the charge domain.

The purpose of this PR is to add these documents to this repository.

Some of these documents have been added as is, some have been altered to the new reality. A few is being discussed at the moment. This PR will not consider changes caused by current discussions, like the model mappings but simply add the documents as is so that we have our current state until a new decision is made.

The following documentation have been added: 
* Guide on how to deploy to your own resource group from your local PC using the terraform code in this repository
* Guide on how to install a SQL Server for local development and make sure you are using the same settings as needed in this domain
* Guide on how to update the template used to install a SQL Server, should the need arise
* Documents for how to handle Mapping into CIM-structures based on eBiX RSM-033 (Change of Charges) and RSM-034 (Notification about charges)

### CODE REVIEW

**Review by prtandrup — APPROVED** [review]
LGTM

File: build/infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Infrastructure as code
+
+## Developer Experience
+
+It is possible to deploy infrastructure from `localhost`.
+
```
Comment (HenrikSommer) [inline build/infrastructure/README.md:6]: A reference to a "Prerequisites" section will make it easier for a first time user.

File: build/infrastructure/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+### `spn_object_id`
+
+This variable has to be filled with the object id of the user that has access to the resource group you want to deploy to.
+
+So if you are deploying to your own resource group from your own machine, this has to be the object id of the user you use to login to the Azure Portal to see the resource group in question.
+
+If the Object ID is unknown, it can be found with using `powershell`.
+
+First, login to the Azure AD with the the account that has access to the resource group. Run the following command and supply your login information:
+
```
Comment (HenrikSommer) [inline build/infrastructure/README.md:93]: "Prerequisites" section:

File: build/infrastructure/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+## Installing `Azure AD for powershell`
+
+Run your `powershell` as an administrator.
+
+The following command installs the module:
+
+```PowerShell
+Install-Module -Name AzureAD
+```
```
Comment (HenrikSommer) [inline build/infrastructure/README.md:120]: One more prerequisite: _Azure Functions Core Tools_
Guide and download:
https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash#v2

File: build/infrastructure/README.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Then, ask the Azure AD for your object ID by using the following command:
+
+```PowerShell
+Get-AzureADUser -searchstring "username@yourdomain.com"
+```
+
+The object ID should now be displayed as the first column in the result.
+
+This ID is the value you need for `object_id` (hint: it looks like a `GUID`)
+
```
Comment (HenrikSommer) [inline build/infrastructure/README.md:107]: "Prerequisites" section:

**Review by HenrikSommer — APPROVED** [review]
I've added a few elements to infrastructure README and added Azure Function publish guide.

**Pull request merged**